src/main/java/frc/robot/commands/drivetrain/ResetGyro.java

```diff
@@ -29,7 +31,12 @@ public void initialize() {
 
     @Override
     public boolean isFinished() {
-        return (Timer.getFPGATimestamp() - startTime) >= delay;
+        if (Timer.getFPGATimestamp() - startTime >= delay) {
```

Comment:
We shouldn't use the timeoutCounter here, this command is meant to time out.

Reply:
Done

Reply:
can you also delete the timeoutCounter?

Reply:
Done